todo-app/src/Tasks.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Description from './Description';
+import Done from './Done';
+import './tasks.css';
+
+export default class Tasks extends React.Component {
```

Comment:
Why did you use plural `Tasks` here? Why not just `Task`? This component receives a task object right? What do you think?

Reply:
Ya you are right, I need to improve naming skills